api/migrations/20210812174149-create-channels.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
       },
       guildId: {
         allowNull: false,
-        type: Sequelize.STRING,
+        type: Sequelize.INTEGER,
```

Comment:
In create-guild.js, you have guildId as string, but here you are changing it to integer.

Reply:
I don't know how I missed that!!! Thanks :)